tests/integration/test_logcollector/test_location/test_location.py

```diff
@@ -0,0 +1,182 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import datetime
+import os
+import sys
+import tempfile
+from shutil import rmtree
+
+import pytest
+from wazuh_testing import logcollector
+from wazuh_testing.logcollector import LOG_COLLECTOR_GLOBAL_TIMEOUT
+from wazuh_testing.tools import LOG_FILE_PATH
+from wazuh_testing.tools.configuration import load_wazuh_configurations
+from wazuh_testing.tools.monitoring import AGENT_DETECTOR_PREFIX, LOG_COLLECTOR_DETECTOR_PREFIX
+from wazuh_testing.tools.monitoring import FileMonitor

```

Comment:
```suggestion
from wazuh_testing.tools.monitoring import AGENT_DETECTOR_PREFIX, LOG_COLLECTOR_DETECTOR_PREFIX, FileMonitor
```

Reply:
Fixed in 0ab8c41819de47e71242dc1303b1ff76bab3103c